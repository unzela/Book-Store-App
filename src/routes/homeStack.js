import { createStackNavigator } from "react-navigation-stack";
import {createAppContainer} from "react-navigation";
import Login from "../login";
import Gallery from "../gallery";
import Category from "../categories";
import BaseHome from "../base";
import Register from "../register";
import Description from "../description";

const screens = {
BaseHome:
{
screen: BaseHome
},
Register:{
    screen:Register
},
Login: {
    screen: Login
},
Category:
{
    screen:Category
},
Gallery : {
    screen: Gallery
},
BookDetails:
{
screen: Description
},
}
const HomeStack = createStackNavigator(screens);

export default createAppContainer(HomeStack);