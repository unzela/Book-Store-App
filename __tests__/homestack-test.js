import 'react-native';
import React from 'react';
import { createStackNavigator } from "react-navigation-stack";
import {createAppContainer} from "react-navigation";
import homeStack from '../src/routes/homeStack';
import renderer from 'react-test-renderer';

test('Home Stack Snapshot',()=>{
    const snap = renderer.create(
        <homeStack/>
    ).toJSON();

    expect(snap).toMatchSnapshot();
});

test('Home Stack Renderer',()=>{
    renderer.create(
        <homeStack/>
    );

  
});
