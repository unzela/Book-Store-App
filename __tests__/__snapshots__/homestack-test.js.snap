// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Stack Snapshot 1`] = `<homeStack />`;
